packages/snap/src/start.ts

```diff
@@ -0,0 +1,88 @@
+// packages/snap/src/dev.ts
+import { createEventManager, createServer, createStateAdapter, getProjectIdentifier, trackEvent } from '@motiadev/core'
+import path from 'path'
+import { flush } from '@amplitude/analytics-node'
+import { generateLockedData, getStepFiles } from './generate-locked-data'
+import { stateEndpoints } from './dev/state-endpoints'
+import { activatePythonVenv } from './utils/activate-python-env'
+import { identifyUser } from './utils/analytics'
+import { version } from './version'
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+require('ts-node').register({
+  transpileOnly: true,
+  compilerOptions: { module: 'commonjs' },
+})
+
+export const start = async (port: number, disableVerbose: boolean): Promise<void> => {
+  const baseDir = process.cwd()
+  const isVerbose = !disableVerbose
+
+  identifyUser()
+
+  const stepFiles = getStepFiles(baseDir)
+  const hasPythonFiles = stepFiles.some((file) => file.endsWith('.py'))
```

Comment:
side note: this is where we will accelerate the start up when we build first since the build command generates a json file with all we need to create the locked data